public/app/core/logs_model.test.ts

```diff
@@ -240,12 +240,12 @@ describe('dataFrameToLogsModel', () => {
 
     expect(logsModel.series).toHaveLength(2);
     expect(logsModel.meta).toHaveLength(2);
-    expect(logsModel.meta[0]).toMatchObject({
+    expect(logsModel.meta![0]).toMatchObject({
```

Comment:
Using `!` in this case makes sense because it is a test, right?

Reply:
My thinking here is that we expect that we have a length so anything after that should be `!` otherwise the test would fail on the length expect.

Reply:
^^^ does that make sense @tskarhed ?

Reply:
Ah, gotcha!